src/components/person.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { MyContext } from './state';
+import { Person as PersonType } from '../types';
+
+interface PersonComponentProps {
+  pers: PersonType;
+}
+
+export const Person: React.FC<PersonComponentProps> = ({ pers }) => {
+  const { name, sex, born, fatherName, motherName, slug, died } = pers;
+  const { setSelectedPerson, person } = useContext(MyContext);
+  const location = useLocation();
+
+  const motherPerson = person.find(
+    (perso: PersonType) => perso.name === motherName,
+  );
+  const fatherPerson = person.find(
+    (perso: PersonType) => perso.name === fatherName,
+  );
+
+  const isPersonInUrl = location.pathname.includes(slug);
```

Comment:
Code quality issue: You are using `String.prototype.includes` to check if the person's slug is in the URL. This could lead to incorrect results if there are other slugs that contain the person's slug as a substring. It would be more accurate to parse the URL and compare the slug directly.